TheMovieDb/SceneDelegate.swift

```diff
@@ -8,45 +8,55 @@
 import UIKit
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
-
+    
     var window: UIWindow?
-
-
+    
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
         // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-        guard let _ = (scene as? UIWindowScene) else { return }
+        
+        guard let windowScene = (scene as? UIWindowScene) else { return }
+        let window = UIWindow(windowScene: windowScene)
+
+        let viewController = HomeViewController()
+        let presenter = HomeViewPresenterImp()
+        viewController.presenter = presenter
+        presenter.viewHome = viewController
+        let navigation = UINavigationController(rootViewController: viewController)
+
+        window.rootViewController = navigation
+        
+        self.window = window
+        window.makeKeyAndVisible()
```

Comment:
Well done with the dependency injection setup. 🥇 
Let's try to use the Composer design pattern to make this implementation cleaner!